### PR TITLE
mark this project deprecated: depends on...

### PR DESCRIPTION
**Pull request by nickboldt**
mark this project deprecated:
depends on first merging https://github.com/eclipse/che-parent/pull/58/files

Change-Id: I77fcdd27dcd085aadcbda4584e04063ce401e4f9
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
Do not merge before eclipse/che-parent#58 is merged.

**Comment by skabashnyuk** [comment]
-1 answered here https://github.com/eclipse/che-parent/pull/58#issuecomment-399910442

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**
